src/components/molecules/NameYourTemplate/index.ts

```diff
@@ -0,0 +1 @@
+export * from './NameYourTemplate';
```

Comment:
This name is unconventional containing the word `Your`.  `TemplateName` or `TemplateNameField` seem more conventional.

Reply:
ooops I've already merged. I'll change this in a follow up PR. 

I'm not sure about naming it `TemplateName` or `TemplateNameField` though. Since that gives the impression it's the input component but this component is for that bit of guidance block just above the actual field.

Reply:
Good point, maybe something like `TemplateNameHint` or `TemplateNameGuidance` etc